apps/communications/contacts/js/activities_vcard.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+/* global LazyLoader, VCardReader, Contacts */
+/* exported vCardActivityHandler */
+
+var vCardActivityHandler = function() {
```

Comment:
nit: as this is a module I would suggest we call this object VcardActivityHandler